client/engine/engine.go

```diff
@@ -267,8 +266,9 @@ func (e *Engine) attemptProgress(objective protocols.Objective) (outgoing Object
 		outgoing.CompletedObjectives = append(outgoing.CompletedObjectives, crankedObjective)
 
 		// Whenever a direct funding objective completes we want to create a consensus_channel
+		// Here we assume that every directfund.Objective is for a ledger channel.
```

Comment:
Is it appropriate at this point to rename `directfund` to something like `createledger`?

A more generic `directfund` would probably be a from-scratch effort anyhow.